### ISSUE TITLE
update links to Understanding WCAG 2.1 -> 2.2

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wai-website#777]
### Resource URL

https://www.w3.org/WAI/media/av

### Description

Throughout all the sub-pages

### Resource Shortname

wai-media-guide

**Comment by remibetin** [comment]
@shawna-slh Want me to help with this? I can easily search & replace in the concerned pages.

**Comment by shawna-slh** [comment]
yes, please! Please also update the footer "Date: Updated..." date, including for the first page (even though that particular page didn't change, I'd like the first page communicate that the overall resource has been updated a bit :-)

**Comment by remibetin** [comment]
@shawna-slh Done!
